### ISSUE TITLE
tests: `test_ls.py` is consistently failing

### ISSUE THREAD

**Issue by skshetry** [iterative/dvc#3893]
```sh
======================================================================== short test summary info ========================================================================
FAILED tests/func/test_ls.py::test_ls_repo - AssertionError: assert {('data', Fal...e.xml', True)} == {('.gitignore....dvc', False)}
FAILED tests/func/test_ls.py::test_ls_repo_with_color - AssertionError: assert '\x1b[01;34md...re.xml\x1b[0m' == '.gitignore\n...ml.dvc\x1b[0m'
FAILED tests/func/test_ls.py::test_ls_repo_recursive - AssertionError: assert {('data/subco...e.xml', True)} == {('.gitignore..., False), ...}
FAILED tests/func/test_ls.py::test_ls_repo_with_removed_dvc_dir - AssertionError: assert {('dep', True...'out', False)} == {('.gitignore...t.py', False)}
FAILED tests/func/test_ls.py::test_ls_repo_with_removed_dvc_dir_recursive - AssertionError: assert {('dep', True.../file', True)} == {('.gitignore...t.py', False)}
FAILED tests/func/test_ls.py::test_ls_remote_repo - AssertionError: assert {('data', Fal...e.xml', True)} == {('.gitignore...dvc', False)}
FAILED tests/func/test_ls.py::test_ls_remote_repo_recursive - AssertionError: assert {('data/subco...e.xml', True)} == {('.gitignore..., False), ...}
FAILED tests/func/test_ls.py::test_ls_remote_repo_with_rev - AssertionError: assert set() == {('.gitignore...odel', False)}
FAILED tests/func/test_ls.py::test_ls_remote_repo_with_rev_recursive - AssertionError: assert {('data/subco...e.xml', True)} == {('.gitignore...', True), ...}
```

Most likely the issue is #3880.

**Comment by pmrowla** [comment]
I think this is fixed by #3883

**Comment by pmrowla** [comment]
`ls` tests are passing in master after #3883 merge (but CI still failing due to unrelated flaky ssh tests)
